routes/cart.route.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const router = express.Router();
+const Cart = require("../models/cart.model")
+
+// GET
+router.get('/', async(req ,res)=>{
+
+  try{
+    const prd = await Cart.find()
+    res.json(prd)
+  }catch(err){
+    console.log("err", err)
```

Comment:
Can send error  response with status code with json like-

res.status(400).json({success:true,message:"error while fetching the product", error:err.message})

Reply:
Thank you, much better now.